### PR TITLE
pandas_issue5

### PR DESCRIPTION
**Pull request by Harveyt47**
Hi @jstac , I have added labels and titles to the figures.

### CODE REVIEW

File: source/rst/pandas.rst [hunk 1]
```diff
@@ -254,7 +254,9 @@ For example, we can easily generate a bar plot of GDP per capita
 
 .. code-block:: python3
 
-    df['GDP percap'].plot(kind='bar')
+    ax = df['GDP percap'].plot(kind='bar')
+    ax.set_xlabel('country', fontsize=12)
+    ax.set_ylabel('GDP per capita ($\\$$)', fontsize=12)
```
Comment (jstac) [inline source/rst/pandas.rst]: I think you can replace `$\\$$` with `\$` and same for below.

File: source/rst/pandas.rst [hunk 2]
```diff
@@ -376,7 +380,9 @@ We can also plot the unemployment rate from 2006 to 2012 as follows
 
 .. code-block:: python3
 
-    data['2006':'2012'].plot()
+    ax = data['2006':'2012'].plot(title='US Unemployment Rate', legend=False)
+    ax.set_xlabel('year', fontsize=12)
+    ax.set_ylabel('$\\%$', fontsize=12)
```
Comment (jstac) [inline source/rst/pandas.rst]: This can be just `\%`.

It's not a maths symbol, just ordinary text.

File: source/rst/pandas.rst [hunk 3]
```diff
@@ -411,7 +417,8 @@ The next code example fetches the data for you and plots time series for the US
     ind = govt_debt.index.droplevel(-1)
     govt_debt.index = ind
     ax = govt_debt.plot(lw=2)
-    plt.title("Government Debt to GDP (%)")
+    ax.set_xlabel('year', fontsize=12)
+    plt.title("Government Debt to GDP ($\\%$)")
```
Comment (jstac) [inline source/rst/pandas.rst]: same here.

**Comment by Harveyt47** [comment]
Hi @jstac , I have decided to delete the $'s and \ for now until we know for sure it renders into the webpage and pdf. We have kept the % as we know they render as raw text on the webpage and in pdf.

**Comment by jstac** [comment]
Roger that, thanks @Harveyt47

**Pull request merged**